code/chapters/index.R

```diff
@@ -1,5 +1,5 @@
 ## ----index-1, echo=FALSE-------------------------------------------------
-is_on_travis = identical(Sys.getenv("TRAVIS"), "true")
+is_on_ghactions = identical(Sys.getenv("TRAVIS"), "true")
```

Comment:
@Robinlovelace this code and the one below need to be updated.

Reply:
Sorted now I think. Take a look and let me know if there are any other things that need changing before we say goodbye to our good friend Travis.